.github/workflows/deploy-on-label.yml

```diff
@@ -61,8 +61,9 @@ jobs:
     - name: Circle CI Deployment Trigger
       id: curl-circle-ci
       if: steps.get-pr-state.outputs.STATE == 'success'
-      uses: Open-Source-Contrib/circle-ci-trigger-action@latest
+      uses: promiseofcake/circleci-trigger-action@v1
```

Comment:
[This new action](https://github.com/promiseofcake/circleci-trigger-action) utilizes the V2 API and already has the capability to support passing in pipeline parameters as JSON via the `payload` attribute.